example/App.js

```diff
@@ -16,45 +21,113 @@ export default class AppContainer extends React.Component {
 
 @connectActionSheet
 class App extends React.Component {
+  state = {
+    selectedIndex: null,
+  };
+
+  _updateSelectionText = (selectedIndex: number) => {
+    this.setState({ selectedIndex });
+  };
+
+  _renderButton = (title: string, onPress: () => void) => (
+    <View style={{ marginBottom: 12 }}>
+      <Entypo.Button
+        name="code"
+        backgroundColor="#3e3e3e"
+        onPress={onPress}>
+        <Text style={{ fontSize: 15, color: '#fff' }}>{title}</Text>
+      </Entypo.Button>
+    </View>
+  );
+
+  _renderSelectionText = () => {
+    const { selectedIndex } = this.state;
+    const text = selectedIndex === null
+      ? 'No Option Selected'
+      : `Option #${selectedIndex + 1} Selected`;
+
+    return (
+      <View style={{ marginTop: 20 }}>
+        <Text style={{ textAlign: 'center', color: 'blue', fontSize: 16 }}>
+          {text}
+        </Text>
+      </View>
+    )
+  };
+
   render() {
     return (
-      <View style={styles.container}>
-        <View style={{ marginBottom: 30 }}>
-          <Text style={{ textAlign: 'center' }}>
-            Hello! This is a simple example app to demonstrate @expo/react-native-action-sheet.
+      <View style={{ flex: 1 }}>
+        <ScrollView style={styles.scrollView} contentContainerStyle={styles.contentContainer}>
```

Comment:
any reason we need a scroll view to render buttons?

Reply:
Yeah it’s in case there are too many options to fit, as it has a max height. This was put in by the previous person who made the original PR.

I can remove it, I just need some direction on how we would handle options that won’t fit on shorter devices.